draft-ietf-scitt-architecture.md

```diff
@@ -87,31 +89,31 @@ It achieves this goal by enforcing the following complementary security guarante
+The guarantees and techniques used in this document generalize those of Certificate Transparency {{-CT}}, which can be re-interpreted as an instance of this architecture for the supply chain of X.509 certificates. However, the range of use cases and applications in this document is much broader, which requires much more flexibility in how each Transparency Service implements and operates its Registry. Each service may enforce its own policy for authorizing entities to register their Signed Statements on the Transparency Service. Some Transparency Services may also enforce access control policies to limit who can audit the full Registry, or keep some information on the Registry encrypted. Nevertheless, it is critical to provide global interoperability for all Transparency Services instances as the composition and configuration of involved supply chain entities and their system components is ever-changing and always in flux.
 
-A TS provides visibility into claims issued by supply chain entities and their sub-systems. These claims are called Digital Supply Chain Artifacts (DSCA).
-A TS vouches for specific and well-defined metadata about these DSCAs. Some metadata is selected (and signed) by the issuer, indicating, e.g., "who issued the DSCA" or "what type of DSCA is described" or "what is the DSCA version"; whereas additional metadata is selected (and countersigned) by the TS, indicating, e.g., "when was the DSCA registered in the Registry". The DSCA contents can be opaque to the TS, if so desired: it is the metadata that must always be transparent in order to warrant trust.
+A Transparency Services provides visibility into Signed Statements originally created as Statements and issued as Signed Statements by supply chain entities and their sub-systems. These Signed Statements (and corresponding Statement payload) are about the objects produced by supply chain objects: Artifacts.
+A Transparency Service vouches for specific and well-defined metadata about these Artifacts that is captured in Statements. Some metadata is selected (and signed) by the Issuer, indicating, e.g., "who issued the Statement" or "what type of Artifact is described" or "what is the Artifact's version"; whereas additional metadata is selected (and countersigned) by the Transparency Services, indicating, e.g., "when was the Signed Statement about the Artifact registered in the Registry". A Statements payload content typically is opaque to the Transparency Services, if so desired: it is the metadata that must always be transparent in order to warrant trust for later processing.
```

Comment:
+1 to the more expressive, expanded acronyms.

>"when was the Signed Statement about the Artifact registered in the Registry" 

Which term do we wish to settle on, and under what context? This is the question under #7 
Personally, I struggle with Registry as a generic term and Transparency Service, which I think confuses transparent with public. 
Either way, the use of registry and transparency service seem interchangeable here. Or, are we talking about different sub-components?

Reply:
This PR intends to do three things (and is not complete in doing it yet):

1. using Statement, Signed Statement & Transparent Statement
2. expanding TS to Transparency Services
3. updating Figure 1, aligning with 1. and 2.

The issues you raise are all valid, but I would like to get 1-3 done in this PR and get another one going for others (if possible).